app/src/main/res/menu/map.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <item
```

Comment:
Czy to menu i w ogóle cały przycisk opcji na toolbarze na aktywności mapy będzie nam potrzebny?